code/obj/submachine/cooking.dm

```diff
@@ -1573,53 +1576,63 @@ TYPEINFO(/obj/submachine/mixer)
 		user.u_equip(W)
 		W.set_loc(src)
 		W.dropped(user)
+		tgui_process.update_uis(src)
+
+	ui_interact(mob/user, datum/tgui/ui)
+		ui = tgui_process.try_update_ui(user, src, ui)
+		if(!ui)
+			ui = new(user, src, "MixerMachine")
+			ui.open()
+	ui_static_data(mob/user)
+		. = list("maxItems" = MIXER_MAX_CONTENTS)
+
+	ui_data(mob/user)
+		var/mixerContents = list()
+		var/index = 1
+		for(var/obj/item/_Item in src.contents)
```

Comment:
What's with the `_Uppercase` naming convention? Normally we don't care too much about naming conventions, but `_` usually indicates something is unused so it's odd.

Reply:
Yeah, not sure what I was thinking, renamed the variable